tests/Feature/Service/ServiceWorkersPageTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tests\Feature\Service;
+
+use Tests\StoreTestCase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use App\AdminUser;
+
+
+class ServiceWorkersPageTest extends StoreTestCase
+{
+    use DatabaseMigrations;
+
+    /** @var AdminUser $adminUser */
+    private $adminUser;
+    private $workersRoute;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->adminUser = factory(AdminUser::class)->create();
+        $this->workersRoute = route('admin.centreusers.index');
+    }
+
+    /**
+     * @test
+     *
+     * @return void
+     */
+    public function itShowsAListOfWorkers()
```

Comment:
This shows a table with headers, but not necessarily users in that list.

I'd suggest setUp() creates a 
- 3 centres in an area
- 1 user with with a homeCentre and no alternatives
- 1 user with with a homeCentre and 2 alternatives
- 1 user with a homeCentre who can Download.

StoreTestCase::seeInElementAtPos() is a custom function available like seeElement() but it can check that elements are visible in a set of elements that are pulled out of a page using a selector. We can use it to check that an array of elements has specific element and contents at an index.

Reply:
@charlesstrange2 I've updated this test based on your feedback :point_up: Can you have a look and let me know if it's ok? You might need to re-approve the PR. :croissant: